packages/core/frontend/sirius-components-omnibox/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@eclipse-sirius/sirius-components-omnibox",
+  "author": "Eclipse Sirius",
+  "version": "2024.9.0",
+  "license": "EPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-sirius/sirius-web"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com/"
+  },
+  "main": "./dist/sirius-components-omnibox.umd.js",
+  "module": "./dist/sirius-components-omnibox.es.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "require": "./dist/sirius-components-omnibox.umd.js",
+      "import": "./dist/sirius-components-omnibox.es.js"
+    }
+  },
+  "scripts": {
+    "start": "vite build --mode 'development' && tsc",
+    "build": "vite build && tsc",
+    "format": "prettier --write \"src/**/*.{js,ts,tsx,css}\"",
+    "format-lint": "prettier --list-different \"src/**/*.{js,ts,tsx,css}\"",
+    "publish:local": "yalc push"
+  },
+  "peerDependencies": {
+    "@eclipse-sirius/sirius-components-core": "*",
```

Comment:
Since it depends on `sirius-components-core` it needs to depend on `@xstate/react` and `state` too

Reply:
Does all `peerDependencies` should be present in this `package.json`?